test/controllers/api/v1/ping_thing_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class PingThingControllerTest < ActionDispatch::IntegrationTest
+  include ResponseHelper
+
+  def setup
+    @user = create(:user)
+    @params_sample = {
+      amount: "1",
+      time: "234",
+      signature: "5zxrAiJcBkAHpDtY4d3hf8YVgKjENpjUUEYYYH2cCbRozo8BiyTe6c7WtBqp6Rw2bkz7b5Vxkbi9avR7BV9J1a6s",
+      transaction_type: "transfer"
+    }
+  end
+
+  test "request without token should return error" do
```

Comment:
avoid using should, same in other tests
```suggestion
  test "request without token returns error" do
```